Resources/Prototypes/Entities/Objects/Materials/Sheets/other.yml

```diff
@@ -107,9 +107,6 @@
   name: plastic
   suffix: Full
   components:
-  - type: Tag
-    tags:
-    - Plastic
```

Comment:
Is the plastic tag not used for anything? Why not just add the `SheetOtherBase` tags to this prototype?

Reply:
good point